components/button/Button.html

```diff
@@ -0,0 +1,222 @@
+              style="
+                width: 800px;
+                height: 150px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </code>
+
+          <hr class="example-hr" />
+          <h2>Example of icon buttons</h2>
+          <p class="example-desc">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo maxime
+            sed alias distinctio ducimus quis.
+          </p>
+
+          <div class="example-showcase">
+            <button class="btn btn-icon">
+              <i class="fas fa-paper-plane"></i>
+            </button>
+            <button class="btn btn-icon-rounded">
+              <i class="far fa-comment"></i>
+            </button>
+          </div>
+
+          <code>
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%2843%2C43%2C43%2C1%29&t=seti&wt=none&l=htmlmixed&width=800&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520btn-icon%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-paper-plane%2522%253E%253C%252Fi%253E%250A%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-icon-rounded%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522far%2520fa-comment%2522%253E%253C%252Fi%253E%250A%253C%252Fbutton%253E"
+              style="
+                width: 800px;
+                height: 169px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </code>
+
+          <hr class="example-hr" />
+          <h2>Link Button</h2>
+          <p class="example-desc">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo maxime
+            sed alias distinctio ducimus quis.
+          </p>
+
+          <div class="example-showcase">
+            <button class="btn btn-link"><a href="#">Link</a></button>
+          </div>
+
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%2843%2C43%2C43%2C1%29&t=seti&wt=none&l=htmlmixed&width=800&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cbutton%2520class%253D%2522btn%2520btn-link%2522%253E%253Ca%2520href%253D%2522link%2522%253ELink%253C%252Fa%253E%253C%252Fbutton%253E"
+            style="
+              width: 800px;
+              height: 76px;
```

Comment:
I have tried carbon but don't like it. If you can use highlight.js or if you go with carbon add every iframe width: 100% and default hight. its make much more responsive.

Reply:
Ya carbon causes issues sometimes, thanks for the idea, will make changes.